src/lib/tls/tls13/tls_psk_identity_13.h

```diff
@@ -40,16 +40,12 @@ class BOTAN_PUBLIC_API(3, 1) PskIdentity {
       /**
        * Construct from an externally provided PSK in the client
        */
-      PskIdentity(PresharedKeyID identity) :
-            m_identity(std::move(identity.get())),
-
-            // RFC 8446 4.2.11
-            //    For identities established externally, an obfuscated_ticket_age of
-            //    0 SHOULD be used, and servers MUST ignore the value.
-            m_obfuscated_age(0) {}
```

Comment:
We moved away from the strong type here (`PresharedKeyID`). Though, I'm still somewhat on the fence whether or not to re-introduce it for clarity.